types/ovenplayer/index.d.ts

```diff
@@ -449,22 +458,4 @@ interface OvenPlayerTrack {
     name: string;
 }
 
-export {
-    OvenPlayerBrowser,
-    OvenPlayerConfig,
-    OvenPlayerEvents,
-    OvenPlayerHandler,
-    OvenPlayerIceServer,
-    OvenPlayerInstance,
-    OvenPlayerPlayList,
-    OvenPlayerQuality,
-    OvenPlayerSource,
-    OvenPlayerState,
-    OvenPlayerTrack,
-    OvenPlayerWebRTCStream,
-};
-
-declare const OvenPlayer: OvenPlayer;
-
-export as namespace OvenPlayer;
```

Comment:
I unfortunately didn't notice this line until now; this would imply that the package is a UMD package. Is that the case? That when loading it into a browser as a script, a global named `OvenPlayer` is declared?

If so, then this package will unfortunately need to be structured a little differently...

Reply:
Yes, it is packaged as UMD for distributing. From source it does not have those global declarations, but am I hearing correctly that this needs to be fixed for consumption as a global?

Can you point me towards some docs for background?

Reply:
```
declare global {
    namespace OvenPlayer {
   }
}

export = OvenPlayer;
```

Is this a close approximation to how it should be structured instead? Instead of moving all contents out of the namespace, I would do the inverse and use `global`

Reply:
No, you'd want to write things like this:

```ts
namespace OvenPlayer {
	export const foo: ...;
}

export as namespace OvenPlayer
export = OvenPlayer;
```

I would recommend looking at other examples of `export as namespace` within this repo.

Reply:
Alright! Seems mostly working.

I am getting a lint failure:

```
Instead of `export =`-ing a namespace, use the body of the namespace as the module body  export-just-namespace
```

However addressing this with the following breaks the exports from the namespace:

```
export as namespace OvenPlayer;

export namespace OvenPlayer {
  ...
}
```

It looks like other examples are disabling `@definitelytyped/export-just-namespace`. Does this package need to do the same?

Reply:
You definitely don't want to export the namespace; maybe it's sufficient to have this PR as-is and just write `export as namespace OvenPlayer` at the bottom...

Reply:
5aa0f0e

Let me know if there's any validation to be done. Not sure how to test typings in a global context.

Reply:
And thanks for the help here.